packages/tldraw/src/lib/ui/context/actions.tsx

```diff
@@ -238,13 +313,17 @@ export function ActionsProvider({ overrides, children }: ActionsProviderProps) {
 						getExportName(editor, defaultDocumentName)
 					)
 				},
+				enabled() {
+					return true
```

Comment:
doing a random scan here, looking at the `onSelect` fn for this action, shouldn't this be checking `pageHasShapes`?

Reply:
I converted everything 1:1 for now. So if an action had some restrictions before then I kept them, but if I didn't then I didn't add new ones. In this particular case we just don't do anything if we don't have any shapes.

Seems quite related to the discussion of whether we can move some of the restrictions that we currently have in `onSelect` to `enabled`.